### PR TITLE
Add docs for deploying nebari on existing k8s

### LINKED ISSUE #256: [DOC] - How to deploy Nebari on an existing Kubernetes cluster
### Preliminary Checks

- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/nebari-dev/nebari/discussions

### Summary

Create a new how-to-guide. We can take relevant sections from: https://github.com/nebari-dev/nebari-docs/pull/113

### Steps to Resolve this Issue

1.
2.
3.
...

### PR DESCRIPTION
**Pull request by viniciusdc**
## Reference Issues or PRs
closes #256 

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

## Documentation

- Initial migration of [old related documentation](https://docs.qhub.dev/en/stable/source/installation/existing.html) present on qhub.dev 

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

File: docs/docs/how-tos/nebari-kubernets.mdx [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+- Replace `provider: aws` with `provider: local`
+- Replace `amazon_web_services` with `local`
+  - And update the `node_selector` and `kube_context` appropriately
+
+<details>
+
+```
+project_name: <project_name>
+provider: local
```
Comment (iameskild) [inline docs/docs/how-tos/nebari-kubernets.mdx:151]:
```suggestion
provider: existing
```

**Review by iameskild — CHANGES_REQUESTED** [review]
This is looking great @viniciusdc! I just made one minor update 👍

**Review by iameskild — APPROVED** [review]
Thanks @viniciusdc :)

**Review by iameskild — DISMISSED** [review]
Sorry for the premature approval... I have one more comment and I think @pavithraes wants to make sure we update any cross reference link as well 👍

File: docs/docs/how-tos/nebari-kubernets.mdx [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+        }
+    ]
+}
+```
+
+</details>
+
+</TabItem>
+<TabItem value="GCP">
+    This is the custom configuration for GCP
```
[the remark this answers is not in the document]
Comment (viniciusdc) [reply docs/docs/how-tos/nebari-kubernets.mdx]: Thanks @iameskild, I removed the mentions of GCP of now, once we had that tested we can include it

**Comment by viniciusdc** [comment]
Hi @pavithraes, I just updated the reference in the get-started, it would be great if you could do a last look into that

**Review by pavithraes — APPROVED** [review]
Looks great, thanks Vini!

**Pull request merged**